t/porting/known_pod_issues.dat

```diff
@@ -661,9 +661,14 @@ perl5380delta
 perl5381delta
 perl5382delta
 perl5383delta
+perl5384delta
```

Comment:
These additions seem unrelated to the GPL changes.

Reply:
Please see https://github.com/Perl/perl5/pull/23231.  A step was missed during recent releases, it appears.